test/functionalTest/cases/1705_csub_cache_objects/csub_cache_objects.test

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
Spined off this (very looooong :) PR to tech debt issue https://github.com/telefonicaid/fiware-orion/issues/1948